code/game/objects/items/clown_items.dm

```diff
@@ -200,11 +200,8 @@
 
 /obj/item/bikehorn/golden/proc/flip_mobs(mob/living/carbon/M, mob/user)
 	for(M in ohearers(7, get_turf(src)))
-		if(ishuman(M) && M.can_hear())
-			var/mob/living/carbon/human/H = M
-			if(istype(H.ears, /obj/item/clothing/ears/earmuffs))
-				continue
-		M.emote("flip")
+		if(M.can_hear())
```

Comment:
This should check ear protection rather than if they can hear audio

Reply:
I dont get it, its not a weapon, its a meme item. Why would it check ear protection like flashbangs?